build-parent/pom.xml

```diff
@@ -121,6 +121,7 @@
         <dockerImageVersion>${project.version}</dockerImageVersion>
 
         <!-- Helm chart repository -->
```

Comment:
I (Informational): adding in the protocol so a downstream project has the chance to update it to it something https if they are using something like Nexus inside a VPN or project-specific repository.